spec/features/families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "Admin sees all active and inactive families" do
+  describe "when click on the 'All' button" do
```

Comment:
Line 4 can be removed because it is no longer implemented in this feature spec

Reply:
Done in sha: `f92ad52`